components/header/README.md

```diff
@@ -7,8 +7,61 @@ Header
 ```
 <!-- STORY -->
 
+### Usage
 
 
+Simple
+```js
+ <div>
```

Comment:
I don't think we need the anonymous div wrapping around any of the examples, and I personally expected one example instead of all of them listed - this allows us to detail the use of the properties in `propTypes`, and, also provide different examples to showcase what is available instead of listing them all for each type of usage